src/backend/distributed/commands/table.c

```diff
@@ -767,7 +767,11 @@ PreprocessAlterTableStmt(Node *node, const char *alterTableCommand,
 		 * distributed table from local table and we don't want to break user
 		 * experience by asking to add coordinator to metadata.
 		 */
+
+		/* as we're having a table type conversion, dont need warning messages */
+		InTableTypeConversionFunctionCall = true;
```

Comment:
Could you please move that flag to `ConvertPostgresLocalTablesToCitusLocalTables` in case we want to use that function anywhere else? `ConvertPostgresLocalTablesToCitusLocalTables` should be responsible for managing that flag instead of caller functions.